From c482786cde2369a9d745bfec535a44d87a0fa76b Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:09:20 -0400
Subject: [usb] core: remove the Kconfig entry for USB_DEBUG

Message-id: <1410887463-20674-489-git-send-email-dzickus@redhat.com>
Patchwork-id: 95525
O-Subject: [RHEL7 PATCH 488/591] usb: core: remove the Kconfig entry for USB_DEBUG
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit a838ec7b0293ba0cdcda9b75cf428c93beea927c
Author: Peter Chen <peter.chen@freescale.com>
Date:   Mon Apr 28 14:12:38 2014 +0800

    usb: core: remove the Kconfig entry for USB_DEBUG

    Since we have already removed the usage of CONFIG_USB_DEBUG, it is
    meaningless that there is still a configuration entry for CONFIG_USB_DEBUG.

    Signed-off-by: Peter Chen <peter.chen@freescale.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/Kconfig b/drivers/usb/core/Kconfig
index db535b0..c3f9a41 100644
--- a/drivers/usb/core/Kconfig
+++ b/drivers/usb/core/Kconfig
@@ -1,13 +1,6 @@
 #
 # USB Core configuration
 #
-config USB_DEBUG
- bool "USB verbose debug messages"
- help
-   Say Y here if you want the USB core & hub drivers to produce a bunch
-   of debug messages to the system log. Select this if you are having a
-   problem with USB support and want to see more of what is going on.
-
 config USB_ANNOUNCE_NEW_DEVICES
  bool "USB announce new devices"
  default N
-- 
1.7.1